apps/modernization-ui/src/apps/patient/add/extended/inputs/administrative/Administrative.module.scss

```diff
@@ -0,0 +1,23 @@
+@use 'styles/colors';
+@use 'styles/borders';
+@use 'styles/tooltip';
+@use 'styles/icons';
+
+.input {
+    @extend %thin;
+    background-color: colors.$base-white;
+
+    header {
+        display: flex;
+        padding: 1.5rem;
+        justify-content: space-between;
+        align-items: center;
+        background-color: colors.$base-white;
+
+        @extend %thin-bottom;
+    }
+
+    .form {
+        background-color: colors.$base-white;
+    }
+}
```

Comment:
suggestion:  This is going to be common to all of the section in this page and could be it's own component.